pkgs/by-name/ii/iio-oscilloscope/package.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  wrapGAppsHook3,
+  curl,
+  fftw,
+  gtk3,
+  gtkdatabox,
+  jansson,
+  libad9361,
+  libiio,
+  libxml2,
+  matio,
+  pcre2,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "iio-oscilloscope";
+  version = "0.17";
+
+  src = fetchFromGitHub {
+    owner = "analogdevicesinc";
+    repo = finalAttrs.pname;
+    rev = "v${finalAttrs.version}-master";
+    hash = "sha256-wCeOLAkrytrBaXzUbNu8z2Ayz44M+b+mbyaRoWHpZYU=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    wrapGAppsHook3
+  ];
+
+  buildInputs = [
+    curl
+    fftw
+    gtk3
+    gtkdatabox
+    jansson
+    libad9361
+    libiio
+    libxml2
+    matio
+    pcre2
+  ];
+
+  env.NIX_CFLAGS_COMPILE = "-Wno-error=maybe-uninitialized";
+  cmakeFlags = [ "-DCMAKE_POLKIT_PREFIX=${placeholder "out"}" ];
+
+  meta = with lib; {
+    description = "Analog Devices' oscilloscope software for Industrial IO (IIO) devices";
+    homepage = "https://wiki.analog.com/resources/tools-software/linux-software/iio_oscilloscope";
+    license = licenses.gpl2;
```

```suggestion
    license = licenses.free;
```
Fairly certain this isn't part of standard GPLv2
>  * - Neither the name of Analog Devices, Inc. nor the names of its
> *   contributors may be used to endorse or promote products derived
> *   from this software without specific prior written permission.